notebooks/03-query/10-functions-ip.ipynb

```diff
@@ -185,6 +185,81 @@
     "display.sql(sql)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d23b9a99-cb5e-4b38-b4f9-07aec117ba5e",
+   "metadata": {},
+   "source": [
+    "### New in Druid 29.0.0 - IPV6_MATCH function\n",
```

Comment:
As elsewhere, remove calling out the specific version that the function relates to.

Instead update the line above to state that this notebook is for version X, and keep the old version with a suffix.

Reply:
Let's discuss versioning again... I though we landed on this. Isn't this the reason to do a separate branch "druid29-preview" so that we release everything when Druid 29.0.0 is ready.

Reply:
Applied versioning strategy including link to new function cell in main README